packages/chain-adapters/src/cosmossdk/thorchain/ThorchainChainAdapter.ts

```diff
@@ -174,6 +175,90 @@ export class ChainAdapter extends CosmosSdkBaseAdapter<KnownChainIds.ThorchainMa
     }
   }
 
+  /* MsgDeposit is used for thorchain swap/lp operations */
+  async buildDepositTransaction(tx: BuildDepositTxInput): Promise<{ txToSign: ThorchainSignTx }> {
+    try {
+      const {
+        wallet,
+        bip44Params = this.defaultBIP44Params,
```

Comment:
Probably shouldn't need to use default bip44Params anymore - these are actually coming from the base adapter but we should be able to send them as part of the input, see https://github.com/shapeshift/lib/pull/1021

Reply:
Disregard the previous comment, let's get this in as is, defaultBip44Params will be removed everywhere in #1021